doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md

```diff
@@ -79,105 +84,141 @@ Let's assume these rules:
                     "entityId",
                     "entityType"
                 ],
-                "regex": "Room\\.(\\d*)Room",
-                "destination": "numeric_rooms",
-                "fiware_service_path": "/rooms"
+                "regex": "other\\.(\\d*)room",
+                "destination": "all_other",
+                "fiware_service_path": "/other"
             },
             {
                 "id": 2,
                 "fields": [
                     "entityId",
                     "entityType"
                 ],
-                "regex": "Room\\.(\\D*)Room",
-                "destination": "character_rooms",
-                "fiware_service_path": "/rooms"
-            },
-            {
-                "id": 3,
-                "fields": [
-                    "entityType",
-                    "entityId"
-                ],
-                "regex": "RoomRoom\\.(\\D*)",
-                "destination": "character_rooms",
-                "fiware_service_path": "/rooms"
-            },
-            {
-                "id": 4,
-                "fields": [
-                    "entityType"
-                ],
-                "regex": "Room",
-                "destination": "other_rooms",
-                "fiware_service_path": "/rooms"
+                "regex": "suite\\.(\\D*)room",
+                "destination": "all_suites",
+                "fiware_service_path": "/suites"
             }
         ]
     }
 
-The above rules set that:
+Now, let's assume the following not-intercepted event regarding a received notification (the code below is an <i>object representation</i>, not any real data format):
+
+```
+notification={
+   headers={
+	   fiware-service=hotel1,
+	   fiware-servicepath=/other,/suites,
+	   correlation-id=1234567890-0000-1234567890
+   },
+   body={
+      {
+	      entityId=suite.12,
+	      entityType=room,
+	      attributes=[
+	         ...
+	      ]
+	   },
+	   {
+	      entityId=other.9,
+	      entityType=room,
+	      attributes=[
+	         ...
+	      ]
+	   }
+	}
+}
+```
+
+As can be seen, two entities (`suite.12` and `other.9`) of the same type (`room`) within the same FIWARE service (`hotel`) but different service paths (`/suites` and `/other`) are notified. `NGSIRestHandler` will crate two `NGSIEvent`'s:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1308/commits/7dbe4ab1ef9d2cc343af43c0e0fe9c6a960dc4af